static/js/people.js

```diff
@@ -884,6 +884,16 @@ export function get_active_human_count() {
     return count;
 }
 
+export function get_active_owner_count() {
+    let count = 0;
+    for (const person of active_user_dict.values()) {
+        if (person.is_owner) {
+            count += 1;
+        }
+    }
+    return count;
```

Comment:
Can we just have this function return whether it is the last owner or not? We currently don't need count of owners in the realm.

Reply:
That makes sense. I've updated for this to `is_current_user_only_owner`, which returns a boolean. I had a question about that new function [here](https://github.com/zulip/zulip/pull/23797/files/e8181d5404ba9170c584b5c82bfa17c05179d41a#r1046179965).